### ISSUE TITLE
FR: Push podspec with latest CocoaPods

### ISSUE THREAD

**Issue by allenhsu** [firebase/firebase-ios-sdk#766]
* Xcode version: 9.2
  * Firebase SDK version: 4.8.2
  * Firebase Component: Auth, Core, Database, Firestore, Messaging, Storage
  * Component version: 4.0.14

#### Steps to reproduce:

I noticed that in the [podspec in master branch](https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseCore.podspec), you've already added `s.static_framework = true`, but it's missing in the corresponding [podspec in CocoaPods/Specs](https://github.com/CocoaPods/Specs/blob/master/Specs/8/b/d/FirebaseCore/4.0.14/FirebaseCore.podspec.json). Is it due to you're committing podspec with pod version < 1.4.0? Is it possible to push podspec to `CocoaPods/Specs` with pod version >= 1.4.0 so `s.static_framework = true` is included, so that we can add dependency for firebase from another private podspec.

**Comment by paulb777** [comment]
@allenhsu - thanks for the question

- Dependencies can still be added onto Firebase pods from other s.static_framework CocoaPods.  Static frameworks can depend upon other static frameworks and/or static library vendored_framework CocoaPods.  The currently published Firebase CocoaPods fall into the latter category.
- We plan to publish Firebase CocoaPods with `s.static_framework = true` and it will be likely be the next Firebase major release since there are some small, but still potentially breaking changes required.

So to summarize, the request you're making is in our plans, but it isn't necessary to add a dependency onto Firebase from another podspec.

**Comment by paulb777** [comment]
Fixed in upcoming 5.0.0 release
